### ISSUE TITLE
Add safe area for iPhone X and above to the bottom menu

### ISSUE THREAD

**Issue by marc-farre** [cuzy-app/humhub-modules-clean-theme#12]
The issue:
![image](https://github.com/cuzy-app/humhub-modules-clean-theme/assets/23310825/6dcf31b6-8b69-4bcd-81ec-3a4df3a7164f)

Solutions tried:
https://github.com/cuzy-app/humhub-modules-clean-theme/commit/5db0feac0c7814a707b324bd00961fbc420ce289#diff-c0399b3002e6de34886f1b5fafffce89cd991e9ef5d5f2f7147d0eaa99d4c0b4R76
And
https://github.com/cuzy-app/humhub-modules-clean-theme/commit/5e1ad7aedf20e8ef98fbd0390a32e00c613b9c40

But it doesn't fix it.
So I had to add 20px of padding bottom for all devices:
https://github.com/cuzy-app/humhub-modules-clean-theme/commit/976d349bd11be6cc1f92002412e67601b37584ab#diff-c0399b3002e6de34886f1b5fafffce89cd991e9ef5d5f2f7147d0eaa99d4c0b4R76

As I don't have access to an iPhone X or later, I cannot test more and fix the issue, because online iPhone simulator doesn't simulate the bottom page correctly.

**Comment by kiwiradler** [comment]
The 20px-fix helps! But, the button for the spaces is still not usable as the window opens downwards and disappears...
![IMG_6404](https://github.com/cuzy-app/humhub-modules-clean-theme/assets/45717431/0eb1be0d-c013-4e64-8d04-30c0e92ebc0f)

**Comment by marc-farre** [comment]
Thanks for the screenshot. I'm sorry, but I cannot help without being able to simulate this.
I've made it test on iPhone 12 mini and the issue doesn't occur on it.

So I think you should revert to version 1.6.0 while this issue isn't fixed: https://github.com/cuzy-app/humhub-modules-clean-theme/releases/tag/1.6.0

**Comment by kiwiradler** [comment]
I've just tested on iPhone 12 mini. The buttons are functional and available - same on iPhone 14. But while the box behind the "hamburger menu" opens upwards (so that all items are reachable), the box for spaces opens downwards and none of the itmes can be reached (see photo above). Can't you just change that, so that it opens upwards as well?

**Comment by marc-farre** [comment]
I could fix it if I can reproduce.

Could you create an account on https://www.cuzy.app/try/ and test it on this HumHub instance (https://demo.cuzy.app/)?
Because on this one, I'me sure there is no other code interfering.
Does the problem occur here too?

On your HumHub instance, have you got the issue only on iPhone or also on Android or mobile simulator such as [Chrome developer tools](https://developer.chrome.com/docs/devtools/device-mode)?

**Comment by kiwiradler** [comment]
With your demo instance, it works perfectly fine. The space menu is not a small box opening up but a complete overlay to the site.

**Comment by ArchBlood** [comment]
Currently did some tests for the following phones with all of these phones having the same issue with the "My Spaces" dropdown not triggering correctly for the upward dropdown menu.

- iPhone X
- iPhone XR
- iPhone 12 Pro
- iPhone 12 Pro Max

I've not had anymore time for test but it should be noted.

**Comment by marc-farre** [comment]
Thanks very much @ArchBlood for testing.
Thanks to your message, I understand we have 2 separated issues:
- This one explained in the issue description which only affects iPhone X and later (should be fixed, but I cannot test)
- The space chooser, which must be a drop up (instead of drop down), when the main navigation bar becomes a bottom menu.

The latter affects all mobiles. Fixed in commit https://github.com/cuzy-app/humhub-modules-clean-theme/commit/fddb562ea08faaaa88b4c99d8eda8c9ece12f950

I've released version 1.7.1 with these fixes: https://github.com/cuzy-app/humhub-modules-clean-theme/releases/tag/1.7.1

Thanks to all for your feedbacks!
